### ISSUE TITLE
successful_checks is always empty for AVIF

### ISSUE THREAD

**Issue by leo-barnes** [gpac/ComplianceWarden#83]
**Describe the issue**
When compiling the latest ComplianceWarden and running it on an AVIF file with JSON output, the "successful_checks" array is always empty.

**Detailed analysis**
From comparing the avif.cpp rules to the av1_hdr10plus.cpp rules, the latter seem to be using `out->covered();`. I'm assuming this is the snippet that marks the rule as successfully checked.

This would be required in order to implement https://github.com/AOMediaCodec/av1-avif/issues/219

**Comment by leo-barnes** [comment]
Yup, the `out->covered();` calls seem to be the issue.

**Comment by rbouqueau** [comment]
Indeed. You can add them based on the code (+ the coverage report you can generate with [this script](https://github.com/gpac/ComplianceWarden/tree/master/scripts/cov.sh)).
